Marlin/UltiLCD2_menu_material.cpp

```diff
@@ -275,12 +275,9 @@ static void lcd_menu_insert_material_preheat()
     int16_t temp = degHotend(active_extruder) - 20;
     int16_t target = degTargetHotend(active_extruder) - 20 - 10;
     if (temp < 0) temp = 0;
-    if (temp > target && temp < target + 20 && !is_command_queued())
+    if (temp > target && temp < target + 20 && (card.pause || !is_command_queued()))
     {
         set_extrude_min_temp(0);
-        for(uint8_t e=0; e<EXTRUDERS; e++)
-            volume_to_filament_length[e] = 1.0;//Set the extrusion to 1mm per given value, so we can move the filament a set distance.
```

Comment:
Changing this factor is a bad idea, if this function is used during a paused print...
...and unnecessary at all, because the volume_to_filament_length is taken into account for all material moves of the change material wizard.